release-notes/7.0/preview/api-diff/rc1/Microsoft.NETCore.App/7.0-rc1_System.Security.Cryptography.md

```diff
@@ -0,0 +1,125 @@
+# System.Security.Cryptography
+
+``` diff
+ namespace System.Security.Cryptography {
+     public abstract class Aes : SymmetricAlgorithm {
+-        public static new Aes Create();
++        [UnsupportedOSPlatformAttribute("browser")]
++        public static new Aes Create();
+     }
+     [ObsoleteAttribute("Derived cryptographic types are obsolete. Use the Create method on the base type instead.", DiagnosticId="SYSLIB0021", UrlFormat="https://aka.ms/dotnet-warnings/{0}")]
+     public sealed class AesCryptoServiceProvider : Aes {
+-        public AesCryptoServiceProvider();
++        [UnsupportedOSPlatformAttribute("browser")]
++        public AesCryptoServiceProvider();
+     }
+-    [ObsoleteAttribute("Derived cryptographic types are obsolete. Use the Create method on the base type instead.", DiagnosticId="SYSLIB0021", UrlFormat="https://aka.ms/dotnet-warnings/{0}")]
+-    public sealed class AesManaged : Aes
++    [ObsoleteAttribute("Derived cryptographic types are obsolete. Use the Create method on the base type instead.", DiagnosticId="SYSLIB0021", UrlFormat="https://aka.ms/dotnet-warnings/{0}")]
++    [UnsupportedOSPlatformAttribute("browser")]
++    public sealed class AesManaged : Aes
+     public abstract class ECDsa : ECAlgorithm {
++        public byte[] SignData(ReadOnlySpan<byte> data, HashAlgorithmName hashAlgorithm);
++        public byte[] SignData(ReadOnlySpan<byte> data, HashAlgorithmName hashAlgorithm, DSASignatureFormat signatureFormat);
++        public int SignData(ReadOnlySpan<byte> data, Span<byte> destination, HashAlgorithmName hashAlgorithm);
++        public int SignData(ReadOnlySpan<byte> data, Span<byte> destination, HashAlgorithmName hashAlgorithm, DSASignatureFormat signatureFormat);
++        public byte[] SignHash(ReadOnlySpan<byte> hash);
++        public byte[] SignHash(ReadOnlySpan<byte> hash, DSASignatureFormat signatureFormat);
++        public int SignHash(ReadOnlySpan<byte> hash, Span<byte> destination);
++        public int SignHash(ReadOnlySpan<byte> hash, Span<byte> destination, DSASignatureFormat signatureFormat);
+     }
+-    [UnsupportedOSPlatformAttribute("browser")]
+-    public class HMACMD5 : HMAC {
++    public class HMACMD5 : HMAC {
```

Comment:
```suggestion
-    [UnsupportedOSPlatformAttribute("browser")]
     public class HMACMD5 : HMAC {
```

Seems like the class should show as context, not diff.  (Since the + and - are identical)

Reply:
Opened https://github.com/dotnet/arcade/issues/10982

Will fix manually.